test/inductor/test_codecache.py

```diff
@@ -334,18 +338,18 @@ def fn(x):
     @parametrize("device", (GPU_TYPE, "cpu"))
     def test_constant_handling(self, device):
```

Comment:
nice. could also parameterize by size and have one < 8 and one >

Reply:
Yeah, can do!